source/Line.cpp

```diff
@@ -455,64 +455,77 @@ Line::initialize()
 
 	// if conditions are ideal, try to calculate initial line profile using
 	// catenary routine (from FAST v.7)
-	real waterDepth = getWaterDepth(r[0][0], r[0][1]);
-	if (r[0][2] == waterDepth) {
-		real XF = dir(Eigen::seqN(0, 2)).norm(); // horizontal spread
+
+	real XF = dir(Eigen::seqN(0, 2)).norm(); // horizontal spread
```

Comment:
These changes seem to break the pendulum test:

https://github.com/FloatingArrayDesign/MoorDyn/actions/runs/5179676881/jobs/9420343210?pr=75